Component/avatar.html

```diff
@@ -0,0 +1,121 @@
+                <a href="modal.html"> Modal</a>
+                <a href="navigation.html"> Navigation</a>
+                <a href="rating.html">Ratings</a>
+                <a href="simplified-grid.html"> Grid</a>
+                <a href="slider.html"> Slider</a>
+                <a href="text-utilities.html"> Typography</a>
+                <a href="toast.html"> Toast</a>
+            </div>
+        </div>
+
+    </div>
+    <div class="row-grid">
+        <div class="grid-col left-nav">
+           
+            <ul class="componentList">
+                <li class="list-item "> <a href="components.html"> Install me</a></li>
+                <li class="list-item"> <a href="alert.html"> Alert</a></li>
+                <li class="list-item active"> <a href="avatar.html"> Avatar</a></li>
+                <li class="list-item"> <a href="badge.html"> Badge</a></li>
+                <li class="list-item"> <a href="button.html"> Button</a></li>
+                <li class="list-item"> <a href="cards.html"> Cards</a></li>
+                <li class="list-item"> <a href="images.html"> Images</a></li>
+                <li class="list-item"> <a href="input.html"> Input</a></li>
+                <li class="list-item"> <a href="lists.html"> Lists</a></li>
+                <li class="list-item"> <a href="modal.html"> Modal</a></li>
+                <li class="list-item"> <a href="navigation.html"> Navigation</a></li>
+                <li class="list-item"> <a href="rating.html">Ratings</a></li>
+                <li class="list-item"> <a href="simplified-grid.html"> Grid</a></li>
+                <li class="list-item"> <a href="slider.html"> Slider</a></li>
+                <li class="list-item"> <a href="text-utilities.html"> Text-Utilities</a></li>
+                <li class="list-item"> <a href="toast.html"> Toast</a></li>
+            </ul>
+        </div>
+        <div class="grid-col">
+            <div class="home-container">
+                <div>
+                    <h2 class="h2-av"> Avatar</h2>
+                    <p class="p-pv"> <span>Avatar </span> are used to represents user presence. <span>Avatar </span> can
+                        be used for multiple purposes. </p>
+                    <span> Sample Avatar </span>
+                </div>
+                <div class="Avatar">
+                    <div class="avatar-all-without-badge">
+                        <img class="micro" src="/Assets/boy.png" alt="micro-Avatar">
+                        <img class="micro" src="/Assets/girl.png" alt="micro-Avatar">
+                        <img class="mini" src="/Assets/boy.png" alt="micro-Avatar">
+                        <img class="mini" src="/Assets/girl.png" alt="micro-Avatar">
+                        <img class="small" src="/Assets/boy.png" alt="micro-Avatar">
+                        <img class="small" src="/Assets/girl.png" alt="micro-Avatar">
+
+                    </div>
+                    <!-- <div class="avatar-all-with-badge">

```

Comment:
You can remove the comments if you are not using it

Reply:
removed the unnecessary commented out things.
Thanks for the suggestions